action_transaction_test.go

```diff
@@ -187,6 +187,37 @@ func Test_RevertTransaction(t *testing.T) {
 	})
 }
 
+func Test_RecordTransaction(t *testing.T) {
+	//var secondTransaction *Transaction
+	t.Run("record transaction", func(t *testing.T) {
```

Comment:
It's difficult to understand what you are testing by only reading the code. It took me a little while to grasp that you are testing hex validation.

Please name the test cases like `argument validation - hex` or `strategy validation - invalid hex` or choose a more descriptive, better name ;-)

Also, in the `given` section, initialize all arguments used for tests, such as `txHex` (its name or value should indicate that it is an invalid argument), `draftId`, etc.

Reply:
Ok. I got it what you mean. Will add details